org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/files/generateResourceService.mtl

```diff
@@ -406,7 +407,7 @@ import [javaClassFullName(aResource, anAdaptorInterface, null) /];
     @Consumes({ MediaType.APPLICATION_FORM_URLENCODED})
     public [creationMethodReturnType(aCreationDialog)/] [creationMethodName(aCreationDialog)/](MultivaluedMap<String, String> formParams
             [commaSeparate(dialogMethodSignature(aCreationDialog, false, true), true, false)/]
-        ) {
+        ) throws URISyntaxException, ParseException {
```

Comment:
Nice one! I will use ParseException in the future myself in similar scenarios, thanks!